src/project/settingup_forms.md

```diff
@@ -309,4 +309,53 @@ To use the drop-down widget in the form from Input:
 
 ![slider](./input_forms_valuerelation1.png)
 ![slider](./input_forms_valuerelation2.png)
-![slider](./input_forms_valuerelation3.png)
\ No newline at end of file
+![slider](./input_forms_valuerelation3.png)
+
+## 1-N relations
+
+It is often the case that you have a set of spatial features and you want to record some parameters every now and then. For example, there is a GIS layer representing the manholes and the surveyors carry out regular inspections of the manholes. Instead of duplicating the manhole layer and recording each inspection, you can create a non-spatial table and store each inspection as a new line. An example project
+
+Another example is when you try to capture multiple photos for a single feature.
+
+In the sections below, you can see how this type of relations can be set up in QGIS and utilised in the Input app.
+
+### Project preparation in QGIS
+
+The example projects can be found:
+- [Assigning multiple inspection to a single feature](<MerginMapsProject id="documentation/forms_one-to-many-relations" />)
+- [Adding multiple photos to a single feature](<MerginMapsProject id="documentation/forms_multiple_photos" />)
+
+To configure 1-N relations in QGIS:
+- From the main menu, select **Projects** > **Properties ...**
+- In the new window, select the **Relations** tab
+- Select **Add Relation** to create a new one
+- A new window will appear, where we can define the parent and child layers and the fields to link the two layers:
+  - For **Name** type **Inspection**
+  - For the **Referenced(parent)** layer, select **manhole_locations**
+  - For **Field 1** of the **Referenced(parent)** layer, select **Manhole**
+  - For the **Referencing(child)** layer, select **inspections**
+  - For **Field 1** of the **Referencing(child)** layer, select **Manhole ID**
+
+![1-N relations in QGIS](./input_forms_many-relations1.png)
+
+This should now allow you to add multiple inspections for each manhole location (**manhole_locations** point layer). The inspections records will be stored in the **inspections** table.
+
+When you open the form for an existing record in the  **manhole_locations** point layer, you should be able to see the existing inspection records and optionally add, delete or edit the records:
+
+![1-N relations in QGIS - form view](./input_forms_many-relations1.png)
+
+The same project when you open add inspection to a manhole in Input will look like the image below:
+
+![Many photos to a single feature](./input_forms_one-to-many.png)
+
```

Comment:
I would suggest to add another header here - .e.g "Link multiple photos to a feature" and add the example project right after that.